API/Controllers/AccountController.cs

```diff
@@ -508,6 +508,19 @@ public async Task<ActionResult> UpdateAccount(UpdateUserDto dto)
             await _userManager.UpdateNormalizedUserNameAsync(user);
             _unitOfWork.UserRepository.Update(user);
         }
+    
+        // Update email
+        if (!string.IsNullOrEmpty(dto.Email) && !user.Email!.Equals(dto.Email))
+        {
+            // Validate email change
+            var emailValidationErrors = await _accountService.ValidateEmail(dto.Email);
+            if (emailValidationErrors.Any()) return BadRequest(await _localizationService.Translate(User.GetUserId(), "email-taken"));
+            user.Email = dto.Email;
```

Comment:
You need a new line after the if statement to help the code be easier to digest.